core/execution/market.go

```diff
@@ -837,6 +837,14 @@ func (m *Market) closeMarket(ctx context.Context, t time.Time) error {
 
 	m.removeOrders(ctx)
 
+	for _, party := range m.liquidity.ProvisionsPerParty().Slice() {
+		// we don't care about the actual orders as they will be cancelled in the book as part of settlement anyways.
+		_, err := m.liquidity.StopLiquidityProvision(ctx, party.Party)
```

Comment:
This returns a list of orders, at this point they most likely are already cancelled right?

Reply:
that's literally what the comment in line 841 says...

Reply:
oh boi... great proof comments are not so useful I guess, or that I'm an idiot.